pathogen_characterisation/data_sources.md

```diff
@@ -1,4 +1,56 @@
 ---
 title: Data sources
-description: Finding and sharing data
+description: Finding and sharing data for pathogen characterisation related data sources.
+contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
+page_id: <!---REPLACE THIS with a shortened page name, with lower case letters and spaces, or an acronym in upper and lower case letters--->
+rdmkit: <!---put the name and URL of the relevant link or links to RDMkit for generic guidelines not specific to infectious diseases--->
+  - name: <!---the name of the RDMkit page--->
+    url: <!---the URL of the RDMkit page--->
+related_pages: 
+  showcase: [<!---REPLACE THIS with the page IDs of the showcase pages that you want to list here as related pages--->]
+  human_biomolecular_data: [<!---REPLACE THIS with the page IDs of the human_biomolecular_data pages that you want to list here as related pages--->]
+  human_clinical_and_health_data: [<!---REPLACE THIS with the page IDs of the human_clinical_and_health_data pages that you want to list here as related pages--->]
+  social_and_economic_impact: [<!---REPLACE THIS with the page IDs of the social_and_economic_impact pages that you want to list here as related pages--->]
+  pathogen_characterisation: [<!---REPLACE THIS with the page IDs of the pathogen_characterisation pages that you want to list here as related pages--->]
+training:
+  - name:
+    registry:
+    url:
 ---
+
+## Introduction
+
+## What is considered as <!--REPLACE THIS with the category that you selected e.g. pathogen characterization--> data?
+
+## Topic 1 <!-- Subsection related to a specific topic related to the data sources of the category that you selected.-->
+
+Short explanation of what this topic is about and why it is important, with an emphasis on infectious diseases and the category that you selected.
+
+### Considerations
+
+Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+
+Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here. Add links to RDMkit in the metadata above, if any are needed. 
+
+### Standards, schemas, ontologies and vocabularies <!--- (optional) --->
+
+List of standards, schemas, othologies and/or vocabularies that are relevant to the resources in this category. Consider having links to the "Data description" page.
```

Comment:
```suggestion
List of standards, schemas, ontologies and/or vocabularies that are relevant to the resources in this category. Consider having links to the "Data description" page.
When applicable, consider adding the appropriate FAIRsharing entry to the tools and resources spreadsheet so that they can be listed and appropriately tagged at the end of the page
```